code/game/objects/items/storage/belt.dm

```diff
@@ -436,34 +436,20 @@
 		))
 
 /obj/item/storage/belt/grenade/full/PopulateContents()
-	new /obj/item/grenade/flashbang(src)
-	new /obj/item/grenade/smokebomb(src)
-	new /obj/item/grenade/smokebomb(src)
-	new /obj/item/grenade/smokebomb(src)
-	new /obj/item/grenade/smokebomb(src)
-	new /obj/item/grenade/empgrenade(src)
-	new /obj/item/grenade/empgrenade(src)
-	new /obj/item/grenade/syndieminibomb/concussion/frag(src)
-	new /obj/item/grenade/syndieminibomb/concussion/frag(src)
-	new /obj/item/grenade/syndieminibomb/concussion/frag(src)
-	new /obj/item/grenade/syndieminibomb/concussion/frag(src)
-	new /obj/item/grenade/syndieminibomb/concussion/frag(src)
-	new /obj/item/grenade/syndieminibomb/concussion/frag(src)
-	new /obj/item/grenade/syndieminibomb/concussion/frag(src)
-	new /obj/item/grenade/syndieminibomb/concussion/frag(src)
-	new /obj/item/grenade/syndieminibomb/concussion/frag(src)
-	new /obj/item/grenade/syndieminibomb/concussion/frag(src)
-	new /obj/item/grenade/gluon(src)
-	new /obj/item/grenade/gluon(src)
-	new /obj/item/grenade/gluon(src)
-	new /obj/item/grenade/gluon(src)
-	new /obj/item/grenade/chem_grenade/incendiary(src)
-	new /obj/item/grenade/chem_grenade/incendiary(src)
-	new /obj/item/grenade/chem_grenade/facid(src)
-	new /obj/item/grenade/syndieminibomb(src)
-	new /obj/item/grenade/syndieminibomb(src)
-	new /obj/item/screwdriver(src)
-	new /obj/item/multitool(src)
+	var/static/items_inside = list(
+		/obj/item/grenade/flashbang = 1,
+		/obj/item/grenade/smokebomb = 4,
+		/obj/item/grenade/empgrenade = 1,
+		/obj/item/grenade/empgrenade = 1,
+		/obj/item/grenade/syndieminibomb/concussion/frag = 10,
+		/obj/item/grenade/gluon = 4,
+		/obj/item/grenade/chem_grenade/incendiary = 2,
+		/obj/item/grenade/chem_grenade/facid = 1,
+		/obj/item/grenade/syndieminibomb = 2,
+		/obj/item/screwdriver = 1,
+		/obj/item/multitool = 1)
+  generate_items_inside(items_inside,src)

```

Comment:
The errors are because this line isn't indented correctly. It should be 1 tab instead of 2 spaces:
```suggestion
	generate_items_inside(items_inside,src)
```
Keep it a proc rather than a macro, it's fine as-is.

Reply:
God dammit. Will fix it now